index.html

```diff
@@ -43,6 +43,185 @@ <h3>LET'S CONNECT</h3>
+                <div class="general_inf">
+                    <p class="year">Canopy&#8226;Backend&#8226;2023</p>
+                </div>
+                <p class="project_description">A daily selection of privately personalized reads; no 
+                    accounts or sign-ups required.</p>
+                <footer>
+                    <ul class="skills">
+                        <li>Html</li>
+                        <li>Css</li>
+                        <li>Javascript</li>
+                    </ul>
+                    <div class="external_link">
+                        <a href="#">see project</a>
+                    </div>
+                </footer>
+            </div>
+            </div>
+        </section>
+        <section>
+            <div class="project">
+            <img class="project_image" src="assets/Snapshoot Portfolio (2).png" alt="">
+            <div class="general">
+                <h3>Tonic</h3>
+                <div class="general_inf">
+                    <p class="year">Canopy&#8226;Backend&#8226;2023</p>
+                </div>
+                <p class="project_description">A daily selection of privately personalized reads; no 
+                    accounts or sign-ups required.</p>
+                <footer>
+                    <ul class="skills">
+                        <li>Html</li>
+                        <li>Css</li>
+                        <li>Javascript</li>
+                    </ul>
+                    <div class="external_link">
+                        <a href="#">see project</a>
+                    </div>
+                </footer>
+            </div>
+            </div>
+        </section>
+        <section>
+        <div class="project">
+            <img class="project_image" src="assets/Snapshoot Portfolio (3).png" alt="">
+            <div class="general">
+                <h3>Multi-post Stories</h3>
+                <div class="general_inf">
+                    <p class="year">Canopy&#8226;Backend&#8226;2023</p>
+                </div>
+                <p class="project_description">A daily selection of privately personalized reads; no 
+                    accounts or sign-ups required.</p>
+                <footer>
+                    <ul class="skills">
+                        <li>Html</li>
+                        <li>Css</li>
+                        <li>Javascript</li>
+                    </ul>
+                    <div class="external_link">
+                        <a href="#">see project</a>
+                    </div>
+                </footer>
+            </div>
+        </div>  
+        </section>
+    </div>
+    
+    <section class="last-sect">
+        <section id="about-myself">
+
+            <div class="myself-abt">
+                <h1>About</h1><h1>Myself</h1>
+                <p id="presentation">I’m a software developer! I can help you build a product , 
+                    feature or website Look through some of my work and experience! 
```

Comment:
- [ ] There should be only one `h1` tag on a file, in your case you have 2. kindly change this to an h2 or h3 as requested by the previous reviewer.

Reply:
@algerina thank you for reviewing, can you please verify my changes after you mentioned.